model/collection.go

```diff
@@ -4,7 +4,17 @@
 package model
 
 type CollectionMetadata struct {
-	Id          string `json:"id"`
-	Name        string `json:"name"`
-	RelativeURL string `json:"url"`
+	Id             string `json:"id"`
+	CollectionType string `json:"collection_type"`
+	TeamId         string `json:"team_id"`
+	Name           string `json:"name"`
+	RelativeURL    string `json:"relative_url"`
+}
+
+type TopicMetadata struct {
+	Id             string `json:"id"`
+	TopicType      string `json:"topic_type"`
+	CollectionType string `json:"collection_type"`
+	TeamId         string `json:"team_id"`
+	CollectionId   string `json:"collection_id"`
```

Comment:
nit, suggested order (Id > TeamId > *Type > Secondary Id)
```suggestion
	Id             string `json:"id"`
	TeamId         string `json:"team_id"`
	TopicType      string `json:"topic_type"`
	CollectionType string `json:"collection_type"`
	CollectionId   string `json:"collection_id"`
```